examples/thrift/build.gradle

```diff
@@ -0,0 +1,56 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+apply plugin: 'application'
+
+description = "Thrift Examples"
+
+startScripts.enabled = false
+
+def grpcVersion = '1.1.0-SNAPSHOT' // CURRENT_GRPC_VERSION
```

Comment:
This also requires an update to our RELEASING.md to add it to the list of files we run sed's on. Search for `examples/build.gradle` in the releasing doc and add this file to each list as well.